styles.css

```diff
@@ -568,3 +536,619 @@ footer {
+    width: 540px;
+    height: 400px;
+    padding-left: 270px;
+    padding-bottom: 40px;
+  }
+
+  .first-flex > .li-cards {
+    list-style: none;
+    display: flex;
+    gap: 10px;
+  }
+
+  .first-flex > .first-flex > .lang-card1 {
+    border: 1px solid #212121;
+    margin-top: 10px;
+    margin-bottom: 20px;
+    padding: 3px;
+  }
+
+  .card1-btn {
+    width: 100px;
+    height: 40px;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    line-height: 24px;
+    background-color: #ff6b00;
+    border: none;
+  }
+
+  .card1-btn:hover {
+    background: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  .card1-btn:active {
+    background: #e05304;
+    color: #f4f1f0;
+  }
+
+  .projects-btn {
+    margin-right: -20px;
+    margin-left: -20px;
+    margin-top: 20px;
+    height: 40px;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    line-height: 24px;
+    background-color: #ff6b00;
+    border: none;
+    display: none;
+  }
+
+  .projects-btn:hover {
+    background: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  .projects-btn1 {
+    margin-right: -20px;
+    margin-left: -20px;
+    margin-top: 20px;
+    height: 40px;
+    color: #fff;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    line-height: 24px;
+    background-color: #ff6b00;
+    border: none;
+  }
+
+  .projects-btn1:hover {
+    background: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  .lang-cards {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 5px 7px;
+    margin-bottom: 10px;
+    color: #fff;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  /** About me SECTION **/
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    margin-top: 80px;
+    background-color: #1c1a19;
+    color: #f4f5f7;
+    background-image: url("img/about-me.png");
+    background-repeat: no-repeat;
+    background-size: 450px;
+    background-position: right 80px;
+  }
+
+  .about-me > .aboutme-flex {
+    display: flex;
+    flex-direction: column;
+    width: 100vh;
+    padding-right: 230px;
+  }
+
+  .about-me > .aboutme-flex > h1 {
+    float: left;
+    color: #ff6b00;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 72px;
+  }
+
+  .about-me > .aboutme-flex > h3 {
+    float: left;
+    color: white;
+    padding-bottom: 5px;
+    width: 327px;
+    text-align: center;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .aboutme-flex > .introduction {
+    float: left;
+    color: white;
+    padding-bottom: 30px;
+    width: 327px;
+    text-align: left;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+  }
```

Comment:
Kindly check and address the dimensions of the about me section since it is pushing out to the left of the page. It should conform to the figma design as much as possible. 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 20-52-13](https://user-images.githubusercontent.com/65233860/193144414-97668a84-f50c-43c4-889c-40b2ee7c8861.png)  |  ![Screenshot from 2022-09-29 20-51-57](https://user-images.githubusercontent.com/65233860/193144431-780de6e6-ca4a-4256-9f72-db534cd03496.png)